helm/aladdin-demo/values.yaml

```diff
@@ -5,6 +5,14 @@ app:
   nginx:
     use: true
     port: 8001
+  liveness:
+    use: true
+    initialDelay: 15
+    period: 120
+  readiness:
+    use: true
+    initialDelay: 5
+    period: 60
 
 deploy:
   # number of seconds for the containers to perform a graceful shutdown, after which it is voilently terminated
```

Comment:
which value are we talking about? 
It may be more obvious if we place the comment on same line as the value.